tests/phpunit/CRM/Mailing/MailingSystemTest.php

```diff
@@ -111,4 +111,22 @@ public function testHtmlWithOpenAndUrlTracking() {
     parent::testHtmlWithOpenAndUrlTracking();
   }
 
+  /**
+   * Test to check Activity being created on mailing Job.
+   *
+   */
+  public function testMailingActivityCreate() {
+    $this->runMailingSuccess([
+      'subject' => 'Example Subject',
+      'body_html' => 'Test Mailing Activity Create',
+      'scheduled_id' => $this->individualCreate(),
+    ]);
+
+    $this->callAPISuccessGetCount('activity', [
+      'activity_type_id' => 'Bulk Email',
+      'status_id' => 'Completed',
+      'subject' => 'Example Subject',
+    ], 1);
```

Comment:
I've seen tests fail when run multiple times in situations like this because after the 1st run there would be more than one activity named "Example Subject".
I like to get around this by doing something like `$subject = uniqid('actCreateTest');`

Reply:
Thanks @colemanw 
Just pushed the change